spec/fixtures/jest/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "description": "",
   "devDependencies": {
-    "jest": "^20.0.0"
+    "jest": "^20.0.0",
+    "mocha": "^3.3.0"
```

Comment:
Could you create a separate fixture folder instead called `spec/fixtures/js_multiple` as I want to keep the existing fixtures to where a single test runner is installed

Reply:
good idea